edit-post/components/header/fullscreen-mode-close/index.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * WordPress Dependencies
+ */
+import { withSelect } from '@wordpress/data';
+import { IconButton, Toolbar } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { addQueryArgs } from '@wordpress/url';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+
+function FullscreenModeClose( { isActive, postType } ) {
+	if ( ! isActive ) {
+		return null;
+	}
+
+	return (
+		<Toolbar className="edit-post-fullscreen-mode-close__toolbar">
+			<IconButton
+				icon="no-alt"
+				href={ addQueryArgs( 'edit.php', { post_type: postType } ) }
+				label={ __( 'Close' ) }
```

Comment:
Since this button is a link, and communicated as such to assistive technologies users, I'd rather consider to use a label that communicates the link destination instead of an action like "Close", which is more appropriate for buttons. RIght now, the link navigates back to the posts list, and it should communicate that 🙂

Reply:
It would work for me 👍 Yeah... I know about the Customizer link 🙈